src/java.desktop/unix/native/libawt_xawt/awt/awt_GraphicsEnv.c

```diff
@@ -572,6 +572,11 @@ getAllConfigs (JNIEnv *env, int screen, AwtScreenDataPtr screenDataPtr) {
 
 cleanup:
     if (success != JNI_TRUE) {
+        for (int i = 0; i < ind-1; i++) {
```

Comment:
New `int i` declaration shadows previously declared `i`, I think it can be reused.

Reply:
I tried building without the new int i and it does seem to be fine. Probably worth it to follow the rest of the class's format and reuse the i var for getAllConfigs.